examples/javascript-object-detection/utils.js

```diff
@@ -0,0 +1,53 @@
+
+// Returns a radom hex color code, i.e. #CD5C5C
+function generateRandomColor(){
+    let maxVal = 0xFFFFFF;
+    let randomNumber = Math.random() * maxVal; 
+    randomNumber = Math.floor(randomNumber);
+    randomNumber = randomNumber.toString(16);
+    let randColor = randomNumber.padStart(6, 0);   
+    return `#${randColor.toUpperCase()}`;
+}
+
+
+function getScaledCoordinates(box, img) {
```

Comment:
I think we can greatly simplify this function. Perhaps using `percentage: true` can help with this too (one of the options to the detector):

![image](https://github.com/xenova/transformers.js/assets/26504141/73fa8990-9734-4d5b-bf9f-6606dc9c2bac)

(see [docs](https://huggingface.co/docs/transformers.js/api/pipelines#objectdetectionpipelinecallimages-options))